Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:16 as builder
+
+COPY ./ /app
+WORKDIR /app
+
+RUN npm install && npm run package
+
+FROM node:16-alpine as runner
```

```suggestion
FROM node:16-alpine
```

You don't need to make alias.